ocf_data_sampler/load/site.py

```diff
@@ -0,0 +1,26 @@
+import pandas as pd
+import xarray as xr
+
+from ocf_data_sampler.config.model import Site
+
+
+def open_site(sites_config: Site) -> xr.DataArray:
```

Comment:
Is this better than just passing two files, one for generation and one for metadata? Site config lugs around quite a lot of extra stuff that's not used here, so it feels cleaner this way, but maybe I'm missing something

Reply:
no, its definitely possible to pass those file names in instead. Sometimes i quite like passing in the well defined object though. I'm pretty 50/50 on this one

Reply:
ill leave it for now